.github/scripts/find-inactive-collaborators.mjs

```diff
@@ -0,0 +1,225 @@
+// Configuration defaults
+const DEFAULTS = {
+  ORG: 'nodejs',
+  TEAM_SLUG: 'nodejs-website',
+  REPO: 'nodejs.org',
+  MONTHS_INACTIVE: 12,
+  ISSUE: {
+    TITLE: 'Inactive Collaborators Report',
+    LABELS: ['inactive-collaborators', 'meta'],
+  },
+};
+
+/**
+ * Gets a date string from X months ago
+ */
+function getDateMonthsAgo(months) {
+  const date = new Date();
+  date.setMonth(date.getMonth() - months);
+  return date.toISOString().split('T')[0];
+}
+
+/**
+ * Checks if a member has had activity since the cutoff date
+ */
+async function isActiveMember(username, cutoffDate, github, repo) {
+  // Check for commits first
+  const commitQuery = `author:${username} repo:${repo} committer-date:>=${cutoffDate}`;
+  const { data: commitData } = await github.rest.search.commits({
+    q: commitQuery,
+    per_page: 1,
+  });
+  if (commitData.total_count > 0) return true;
+
+  // Then check for issue/PR activity
+  const issueQuery = `involves:${username} repo:${repo} updated:>=${cutoffDate}`;
```

Comment:
This technically checks where a PR they were involved in has been updated in the past twelve months, but I think that's okay, right?